js/src/admin/components/AdminPage.tsx

```diff
@@ -40,18 +40,21 @@ export interface HTMLInputSettingsComponentOptions extends CommonSettingsItemOpt
   type: HTMLInputTypes;
 }
 
+const BooleanSettingTypes = ['bool', 'checkbox', 'switch', 'boolean'] as const;
```

Comment:
nit: is there no option for something like "const readonly string[]"?

Reply:
That removes the entire point of `as const`.